app/src/main/java/com/example/bigowlapp/repository/ScheduleRepository.java

```diff
@@ -18,7 +18,7 @@ public ScheduleRepository() {
     }
 
     public Task<Void> updateScheduleMemberResponse(String scheduleId, String userUid, UserScheduleResponse currentUserScheduleResponse) {
-        return collectionReference.document(scheduleId).update("members.".concat(userUid), currentUserScheduleResponse);
+        return collectionReference.document(scheduleId).update("userScheduleResponseMap.".concat(userUid), currentUserScheduleResponse);
```

Comment:
Again, consider using a magic literal.

Reply:
Totally agree, but will leave this for #82 .

Although if you think it's better to use a constant now, just reply that I should do so.

Reply:
Do it anyways.
Just make sure you're commit comment is in this format "[#202] Refactor: Removed Magic Literal; *additional comment*"

Reply:
@ShawnGuerra-Bautista Done!